### ISSUE TITLE
Geoprocessing prepare script - MacOS unable to recursively copy directory with symlink

### ISSUE THREAD

**Issue by twelch** [seasketch/geoprocessing#242]
The geoprocessing `prepare.ts` script recursively copies `base-project` to `geoprocessing/dist` in preparing the bundles

```
    await $`cp -r ${baseProjectPath}/* ${distBaseProjectPath}`;
    await $`cp -r ${baseProjectPath}/. ${distBaseProjectPath}`;
    await $`mv ${distBaseProjectPath}/.gitignore ${distBaseProjectPath}/_gitignore`;
```

Since `base-project/node_modules/@seasketch/geoprocessing`  symlinks to sibling geoprocessing package, the copy command seems to copy the `base-project/node_modules/@seasketch/geoprocessing` symlink, then try to copy files by following the src symlink, into the destination symlink, which leads to a recursive loop between the two symlinks.

**Comment by twelch** [comment]
The `cp` command on macos doesn't have an `exclude directory` options.

One solution is to not have the cp follow symlinks?  Via https://superuser.com/questions/216919/how-to-copy-symlinks-to-target-as-normal-folders

$ mkdir dest_dir
$ cp symlink_dir/* dest_dir/
the drawback is that you have to create the destination directory first

**Comment by twelch** [comment]
Copy command also might offer the right option

Either don't follow the symbolic link on copy
      -P, --no-dereference
              never follow symbolic links in SOURCE

or maybe we do want it to follow, just do it better

-R   Copy the folder and subtree (recursive).  
        If source_file designates a directory, [cp](https://ss64.com/bash/cp.html) copies the directory and
        the entire subtree connected at that point.  If the source_file
        ends in a /, the contents of the directory are copied rather than
        the directory itself.  This option also causes symbolic links to be
        copied, rather than indirected through, and for cp to create special
        files rather than copying them as normal files.  Created
        directories have the same mode as the corresponding source directory,
        unmodified by the process' umask.

        COMPATIBILITY Historic versions of the cp utility had a -r option.
        This implementation supports that option; however, its use is strongly
        discouraged, as it does not correctly copy special files, symbolic links, or fifo's.

        A Homebrew-installed (GNU) cp does not have this limitation.

**Comment by twelch** [comment]
Resolved in 1072d14f6534e696072bbd8586b7eb1bcf540abe
